app/views/revisions/show.html.erb

```diff
@@ -1,23 +1,35 @@
 <% content_for :title, "#{@record.class.model_name} ##{@record.id} - comparing revisions" %>
 
-<% if @node %>
-  <%= render "nodes/sidebar" %>
+<% content_for :breadcrumbs do %>
+  <%= render partial: 'breadcrumbs' %>
+<% end %>
+
+<% content_for :sidebar do %>
+  <% if @issue %>
+    <%= render 'issues/sidebar' %>
+  <% elsif @list %>
+    <%= render 'cards/sidebar' %>
+  <% elsif @node %>
+    <%= render 'nodes/sidebar' %>
+  <% end %>
 <% end %>
```

Comment:
We can possibly turn this into an Ajax call, in a completely separate PR. I think the Ajax PR should probably be done/merged before the rest (breadcrumbs, and Activity tab => View History).